pytorch_lightning/core/optimizer.py

```diff
@@ -350,31 +356,18 @@ def _validate_scheduler_api(lr_schedulers: List[Dict[str, Any]], model: "pl.Ligh
             )
 
 
-def _get_default_scheduler_config() -> Dict[str, Any]:
```

Comment:
Good that flash no longer depends on it https://github.com/PyTorchLightning/lightning-flash/pull/1099
We should be good here 😃

Reply:
Another side comment is that `LRSchedulerConfig` is not protected so I guess we'll have to keep compatibility for it from now on.